.eslintrc.json

```diff
@@ -42,6 +42,7 @@
     "import/prefer-default-export": "off",
     "no-confusing-arrow": "off",
     "arrow-parens": ["warn", "as-needed"],
+    "arrow-body-style": "off",
```

Comment:
This one gave me some strange alert, which prevented code from compiling. I'll be happy to fix it if anyone knows how to satisfy this requirement in code.

Reply:
Can you write more about the problem? What is the error msg? It works fine for me without this line of code.